scripts/spawn_peds.py

```diff
@@ -0,0 +1,549 @@
+# https://github.com/carla-simulator/carla/tree/master/PythonAPI/examples
+import glob
+import os
+import sys
+import time
+import subprocess
+
+sys.path.append('../carla/dist/carla-0.9.9-py3.7-linux-x86_64.egg')
```

Comment:
Path can be injected through properties file / YAML file

Reply:
Will update all this once we setup CARLA

Reply:
yup.. just mentioning this for future reference